FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -51,9 +221,328 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <locus target="#115v"></locus>
+                                <title ref="LIT6354MEMiracle"></title>
+                            </msItem>
+                           <msItem xml:id="p1_i2.5.2">
+                               <locus target="#116r"></locus>
+                               <title ref="LIT6355MEMiracle"></title>
+                           </msItem>
+</msItem>
+                            
+                            <msItem xml:id="p1_i2.6">
+                                <locus from="116rb" to="117rb"></locus>
+                                <title ref="LIT6356MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.7">
+                                <locus from="117rb" to="118ra"></locus>
+                                <title type="complete">Miracle concerning the sickness of a monk</title>
+                                                        </msItem>
+                            
+                            <msItem xml:id="p1_i2.8">
+                                <locus from="118ra" to="119rb"></locus>
+                                <title ref="LIT6358MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.9">
+                                <locus from="119rb" to="119vb"></locus>
+                                <title ref="LIT6359MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.10">
+                                <locus from="119vb" to="120ra"></locus>
+                                <title ref="LIT6360MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.11">
+                                <locus from="120ra" to="120rb"></locus>
+                                <title ref="LIT6362MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.12">
+                                <locus from="120rb" to="121va"></locus>
+                                <title ref="LIT6363MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.13">
+                                <locus from="121va" to="121vb"></locus>
+                                <locus from="123ra" to="123vb"></locus>
+                                <title ref="LIT6365MESick"></title>
+                            </msItem>                            
+                            <msItem xml:id="p1_i2.14">
+                                <locus from="124ra" to="127ra"></locus>
+                                <title ref="LIT6366MEDeath"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.15">
+                                <locus from="127ra" to="127va"></locus>
+                                <title>Miracle No. 15</title>
```

Comment:
ref="LIT6804MiracleEwostWarasina"

Reply:
Added the ref as given by you. Shall I delete the following note?